pkg/operation/shoot/shoot.go

```diff
@@ -244,6 +245,10 @@ func (s *Shoot) GetZones() []string {
 		return s.Info.Spec.Cloud.OpenStack.Zones
 	case gardenv1beta1.CloudProviderAlicloud:
 		return s.Info.Spec.Cloud.Alicloud.Zones
+	case gardenv1beta1.CloudProviderPacket:
+		return &s.Info.Spec.Cloud.Packet.Networks.K8SNetworks
+	case gardenv1beta1.CloudProviderLocal:
```

Comment:
remove local - the local provider is gone

Reply:
Done